include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -139,13 +143,11 @@ struct GemmKernel
         return true;
     }
 
-    CK_TILE_DEVICE void operator()(GemmCommonKargs kargs) const
+    CK_TILE_DEVICE auto make_gemm_tensor_views(const ADataType* a_start,
```

Comment:
Please use coherent function naming style: 
```suggestion
    CK_TILE_DEVICE auto MakeTensorViews(const ADataType* a_start,
```